src/librustdoc/html/render/context.rs

```diff
@@ -635,7 +635,7 @@ impl<'tcx> FormatRenderer<'tcx> for Context<'tcx> {
                     buf,
                     "<div class=\"main-heading\">\
                      <h1 class=\"fqn\">\
-                         <span class=\"in-band\">Rustdoc settings</span>\
+                         Rustdoc settings\
                      </h1>\
```

Comment:
I think we can inline it completely at this point:

```suggestion
                     <h1 class=\"fqn\">Rustdoc settings</h1>\
```